frontend/vue.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  devServer: {
+    proxy: {
```

Comment:
Nice job using a devServer proxy for these!

Reply:
Oh my god I was having so much trouble trying to get the the frontend to make our api calls without running into cors errors.  After hours of reading I finally got this to work.